### PR TITLE
Add index.html and scripts.js

### PR DESCRIPTION
**Pull request by aganova**
Zadanie dot. instrukcji warunkowych

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+	<meta charset="utf-8">
+	<title>Instrukcje warunkowe w praktyce</title>
+	<link rel="stylesheet" href="css/style.css">
```
Comment (matwjablonski) [inline index.html]: type attr is necessary

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	b = prompt("Wpisz wartość b"),
+	value = (a * a) - (2 * a * b) + (b * b);
+alert("Wynik działania (a * a) - (2 * a * b) + (b * b) z następującymi wartościami: a = " + a + " i b = "+ b + " wynosi " + value + ".");
+console.log("Wynik działania (a * a) - (2 * a * b) + (b * b) z następującymi wartościami: a = " + a + " i b = "+ b + " wynosi " + value + ".");
+
+if (value > 0 ) {
+	console.log("Wynik dodatni.");
+} else if (value < 0 ) {
+	console.log("Wynik ujemny.");
+} else {
```
Comment (matwjablonski) [inline js/scripts.js]: are you sure that value in last case will be equal to 0? what will happen when i type a letter inside prompt? :)

File: index.html (hunk 1 above)
Comment (aganova) [reply index.html]: You mean a type attribute in a link tag? Is it really necessary in HTML5? 

https://stackoverflow.com/questions/5409114/is-type-text-css-necessary-in-a-link-tag

File: js/scripts.js (hunk 2 above)
Comment (aganova) [reply js/scripts.js]: It won't work properly. Gosh, what a mistake!! :)

File: index.html (hunk 1 above)
Comment (matwjablonski) [reply index.html]: you are right ;)

**Pull request merged**